pkg/deploy/reaper/reaper.go

```diff
@@ -55,9 +55,21 @@ func (reaper *DeploymentConfigReaper) Stop(namespace, name string, timeout time.
 		return err
 	}
 
-	// If there is neither a config nor any deployments, we can return NotFound.
+	options = kapi.ListOptions{LabelSelector: util.DeployerPodSelector(name)}
```

Comment:
@kargakis i'm working with single deployment here... I guess that will give me all pods from all deployments... do we want to erase them'll ?